src/scales/quantitative.js

```diff
@@ -123,23 +124,50 @@ export function ScaleLog(key, channels, {base = 10, domain = inferLogDomain(chan
   return ScaleQ(key, scaleLog().base(base), channels, {...options, domain});
 }
 
+export function ScaleSymlog(key, channels, {constant = 1, ...options}) {
+  return ScaleQ(key, scaleSymlog().constant(constant), channels, options);
+}
+
 export function ScaleQuantile(key, channels, {
-  quantiles = 5,
+  quantiles,
```

Comment:
Makes me wonder if we should support *n* for quantile scales, too (renaming the *quantiles* option)?